estela-web/src/components/JobDetailPage/index.tsx

```diff
@@ -253,192 +609,620 @@ export class JobDetailPage extends Component<RouteComponentProps<RouteParams>, J
         );
     };
 
-    render(): JSX.Element {
+    getItems = (page: number, pageSize: number): void => {
+        const requestParams: ApiProjectsSpidersJobsDataListRequest = {
+            pid: this.projectId,
+            sid: this.spiderId,
+            jid: this.newJobId,
+            type: "items",
+            page: page,
+            pageSize: pageSize,
+        };
+        this.apiService.apiProjectsSpidersJobsDataList(requestParams).then(
+            (response) => {
+                let data: Dictionary[] = [];
+                if (response.results?.length) {
+                    const safe_data: unknown[] = response.results ?? [];
+                    data = safe_data as Dictionary[];
+                    console.log(data);
+                    this.setState({ items: data, loadedItems: true });
+                }
+                this.setState({ loadedItems: true });
+            },
+            (error: unknown) => {
+                console.error(error);
+                resourceNotAllowedNotification();
+            },
+        );
+    };
+
+    overview = (): React.ReactNode => {
         const {
-            loaded,
-            args,
-            envVars,
             tags,
-            date,
-            status,
             lifespan,
-            totalResponseBytes,
-            created,
-            cronjob,
-            stats,
+            loadedItems,
+            envVars,
+            args,
+            date,
             dataStatus,
-            dataExpiryDays,
-            loading_status,
-            modified,
+            spiderName,
+            totalResponseBytes,
+            requestCount,
+            items,
+        } = this.state;
+        const radius = 100;
+        const circunference: number = ((2 * 22) / 7) * radius;
+        const storageDecimalPercentage = dataStatus !== "DELETED" ? (totalResponseBytes || 0) * 1e-9 : 0;
```

Comment:
Some cases the storage data is really short, and in the circle doesn't show anything. You could display the size even in megabytes to know there is a consume by the job